DEVELOPMENT.md

```diff
@@ -295,41 +389,18 @@ BentoML also provides git hooks that developers can install with:
 make hooks
 ```
 ## Stubs
-Refers to [Installation](https://github.com/microsoft/pyright#installation) to install pyright correctly.
+Refer to [Installation](https://github.com/microsoft/pyright#installation) to install pyright correctly.
 
-In order to make pyright function correctly one also need to run the following scripts alongside with the stubs provided
- in the main repository.
+In order for pyright to function correctly you must also run the following scripts alongside the stubs provided in the main repository.
 
-One can also clone a [copy](https://github.com/bentoml/stubs) of all dependencies stubs used by BentoML to `typings/` via:
+You can also clone a [copy](https://github.com/bentoml/stubs) of all the dependencies stubs used by BentoML to `typings/` via:
 ```bash
 # Assuming at $GIT_ROOT
 git clone git@github.com:bentoml/stubs.git
 \rm -rf typings/.git* typings/*.{sh,toml,txt,cfg,py,md}
 ```
```

Comment:
I forgot to fix this, but I think we should remove this part. Can you help us removing this? Thank you